app/komponenter/spinner/spinner.module.css

```diff
@@ -0,0 +1,6 @@
+.spinnerKonteiner {
+  min-height: 100vh;
```

Comment:
Litt spent på om denne kan gi noen "morsomme" utslag, eller om det går fint. Kommer kanskje an på konteksten der spinneren brukes? 100vh kan jo fort være litt høyt hvis det er noe info nedenfor spinneren?

Reply:
For alle andre som leser PRen;
Vi tok i bruk `70vh`.  Da minimerer man skrolling og spinneren får god plass.

Godt poeng fra Charlie; Spinner skal berre vises maksimalt i et par sekund så det bør gå fint med litt skrolling på enkelte enheter!